bin/beyond

```diff
@@ -28,15 +28,17 @@ versionCheckPromise
       .command('build', 'Rebuild the project for distribution.')
       .command('clean', 'Remove all distribution files.')
       .command('dev', 'Startup a development session.')
+      .command('imp', 'Summon an imp')
```

Comment:
I get the joke, but we will probably need to update this to something more informative :)

Reply:
I've suggested to use a simple `Deprecated` as we agreed to hide a bit this option. After all it's an impersonation

Reply:
Security by obscurity is rarely a good idea though. If someone is not an admin they will not be able to impersonate anyway, so I would leave it visible.

Reply:
Oh... I though no one reads this ;-)